blinky/.tool-versions

```diff
@@ -0,0 +1 @@
+../.tool-versions
```

Comment:
I didn't know this was a thing! Nice! 👍 🚀

Reply:
I _think_ this is just a symlink.

Reply:
Oh. It looked like just a file that has a relative path as its contents instead of listing the actual versions in it, so I was thinking it was an `asdf` thing. Normal symlink makes total sense. 👍